src/server/markup.js

```diff
@@ -1,16 +1,20 @@
-export default (reactString, initialData, webserver) => {
+export default (reactString, initialData, webserver, config) => {
+  const stylesheet = process.env.NODE_ENV === 'production'
+    ? `<link rel='stylesheet' href='${webserver}/dist/styles.css'>`
+    : '';
+
   return (
     `<!doctype html>
-    <html lang='en-us'>
+    <html lang='${config.lang}'>
       <head>
-        <meta charset='utf-8'>
-        <title>Hapi Universal Redux</title>
-        <link rel='shortcut icon' href='/favicon.ico'>
+        <meta charset='${config.charset}'>
+        <title>${config.title}</title>
+        ${stylesheet}
       </head>
```

Comment:
@artibella mentioned we might use [`react-helmet`](https://github.com/nfl/react-helmet) here later on, but we'll see